docs/en/contributing-to-appium/developers-overview.md

```diff
@@ -0,0 +1,149 @@
+0. Update the CHANGELOG and/or README with any appropriate changes and commit. Most subpackages don't have a CHANGELOG.
+0. Run `npm version <version-type>` with the appropriate version type.
+0. This will create a version tag on your branch. Push to the appropriate branch on GitHub.
+0. Run `npm publish` (with `--tag beta` if this isn't an official release).
+
+For the main Appium packages, all the above steps must be taken, but with
+several changes. One reason is that for the main package we use NPM shrinkwrap
+to ensure dependencies don't change between installations. Another reason is
+that we develop on master and release on various branches. The way it works is
+as follows: we always develop and add new code to master. When we are ready to
+make a new minor or major release (i.e., `1.5.0` or `2.0.0`), we create
+a release branch (`1.5` or `2.0` respectively). We then publish off of that
+branch. As we feel the need to make patch releases, we first pull the patch
+into master, then cherry-pick individual patches to the release branch (perhaps
+even multiple release branches). Then we again publish from those branches with
+the updated patch version (`1.5.1` or `2.0.1` for example).
+
+0. Remove the NPM shrinkwrap JSON file.
+0. `rm -rf node_modules && npm install` and run tests to make sure a clean install works.
+0. `rm -rf node_modules && npm install --production` to get just the production deps.
+0. `npm shrinkwrap` to write the new NPM shrinkwrap JSON file.
+0. Determine whether we have a patch (bugfix), minor (feature), or major (breaking) release according to the principles of SemVer.
+0. Update `package.json` with the appropriate new version.
+0. Update the CHANGELOG/README with appropriate changes and submit for review as a PR, along with shrinkwrap and `package.json` changes. Wait for it to be merged, then pull it into the release branch.
+0. Create a tag of the form `v<version>` on the release branch (usually a minor branch like `1.5` or `1.4`), with: `git tag -a v<version>`, e.g., `git tag -a 1.5.0`. This is not necessary for beta versions.
+0. Push the tag to upstream: `git push --tags <remote> <branch>`
+0. Run `npm publish` (with `--tag beta` if this isn't an official release).
+0. Update the docs at appium.io. Check out the appium.io repo from github, check out the `gh-pages` branch and pull latest. Run `rake publish`.
+0. Create a new release on GitHub: go to `https://github.com/appium/appium/releases/tag/v<VERSION>` and hit "Edit Tag". Make the release name `<VERSION>` (e.g., `2.0.5`), then paste in the changelog (but not the changelog header for this version). If it's a beta release, mark as pre-release.
+0. Create a new post on discuss.appium.io announcing the release. Post it in the "News" category. Paste in the changelog and any choice comments. Pin it and unpin the previous release post.
+0. Notify Appium.app and Appium.exe of the new build so that new GUIs can be released.
```

Comment:
Are we releasing new GUIs? appium-dot-app hasn't been touched since `Jul 21, 2015`